### PR TITLE
Chore: Organize testing deployment addresses

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

File: data/config/court.rpc.js [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-  address: undefined                                           // if unset, it will try to fetch it from a previous deploy for devnet
+  address: undefined              // if unset, it will try to fetch it from a previous deploy for devnet rpc
 }
 
 module.exports = {
   governor: {
-    funds:                        '0x0000000000000000000000000000000000000001',
-    config:                       '0x0000000000000000000000000000000000000002',
-    modules:                      '0x0000000000000000000000000000000000000003',
+    funds:                        '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1', // ganache initial deterministic address
```
Comment (bingen) [inline data/config/court.rpc.js]: Which version of ganache? Which mnemonic being used? I don't see that one.

File: package.json [hunk 2]
```diff
@@ -13,13 +13,14 @@
...
+    "deploy:court:rpc": "rm -f data/output/{minime,court}.rpc.json && npm run deploy:dai:rpc && npm run deploy:anj:rpc && node ./bin/index.js court -n rpc",
     "deploy:court:ropsten": "node ./bin/index.js court -n ropsten",
     "deploy:court:rinkeby": "node ./bin/index.js court -n rinkeby",
-    "deploy:court:mainnet": "node ./bin/index.js court -n mainnet"
+    "deploy:court:mainnet": "node ./bin/index.js court -n mainnet",
+    "postinstall": "cd node_modules/@aragon/court && npm i && cp ../../../truffle-config.js truffle-config.js && npm run prepublishOnly"
   },
   "dependencies": {
-    "@aragon/court": "^1.0.0",
+    "@aragon/court": "aragon/aragon-court#improve_emitted_events",
```
Comment (bingen) [inline package.json]: Shouldn't we use a version instead?

File: package.json [hunk 3]
```diff
@@ -13,13 +13,14 @@
     "deploy:anj:ropsten": "node ./bin/index.js minime -n ropsten -c ./data/config/anj.js",
     "deploy:anj:rinkeby": "node ./bin/index.js minime -n rinkeby -c ./data/config/anj.js",
     "deploy:anj:mainnet": "node ./bin/index.js minime -n mainnet -c ./data/config/anj.js",
-    "deploy:court:rpc": "npm run deploy:dai:rpc && npm run deploy:anj:rpc && node ./bin/index.js court -n rpc",
+    "deploy:court:rpc": "rm -f data/output/{minime,court}.rpc.json && npm run deploy:dai:rpc && npm run deploy:anj:rpc && node ./bin/index.js court -n rpc",
```
Comment (bingen) [inline package.json]: Brace expansion is not working for me, not sure if it's a node problem or what. :man_shrugging:

File: package.json (hunk 2 above)
Comment (facuspagnuolo) [reply package.json]: yes, this is wip, I'll replace it with the rc once we have it

File: data/config/court.rpc.js (hunk 1 above)
Comment (facuspagnuolo) [reply data/config/court.rpc.js]: I'll add a ganache dependency here, however, having this is better than the previous one

File: package.json (hunk 3 above)
Comment (facuspagnuolo) [reply package.json]: weird, we can split it then

**Pull request merged**